packages/zoe/src/contractSupport/priceQuote.js

```diff
@@ -23,3 +25,10 @@ export const getAmountIn = quote => getPriceDescription(quote).amountIn;
 export const getAmountOut = quote => getPriceDescription(quote).amountOut;
 /** @type {(quote: PriceQuote) => Timestamp} */
 export const getTimestamp = quote => getPriceDescription(quote).timestamp;
+
+/** @param {Brand<'nat'>} brand */
+export const unitAmount = brand => {
+  const decimalPlaces = brand.getDisplayInfo().decimalPlaces ?? 0;
```

Comment:
brands are typically remote. I doubt this will work.

Reply:
The param is declared `Brand`, not `ERef<Brand>`. Do you mean this function won't work as written or won't be as useful as you're inferring I meant it to be?

Reply:
AmountMath methods take `Brand` and not `ERef` brand, so I thought it wasn't remotable, but the first line of most of them is `assertRemotable`. I take it Brand is always remote, and can't be made present? Or are the types wrong?

6d729a7c0fa9140e1b71514ce4142487c76cccfa

Reply:
Unless you created the brand, you don't know what vat it was created in, so you have to assume it's remote and can only be called with `E()`. I can't think of a good way to capture this in TypeScript, though I gather there attempts rattling around somewhere.

AmountMath use brands for their identity; they don't call methods on the brand. They rely on their caller having resolved any promises to presences. For that purpose, `Promise<Brand>` is no good, hence `ERef<Brand>` is probably not right.